src/UserHome/UserHome.js

```diff
@@ -2,16 +2,14 @@ import React, { Component } from "react";
 import MovieContainer from '../MovieContainer/MovieContainer';
 import { Link } from "react-router-dom";
 
-const UserHome = ({ appState, getUsersRatings}) => {
-	getUsersRatings(appState.userID);
-
+const UserHome = ({ appState, getUsersRatings, changeUserId}) => {
 	return (
 		<>
 			<header className="App-header">
 				<h2>Decaying Ketchup</h2>
 				<h3>Welcome, {appState.userName}!</h3>
 			<Link to={`/`}>
-				<button>Logout</button>
+				<button onClick={() => changeUserId({id: null, userName: null})}>Logout</button>
```

Comment:
added this to reset the App's state when a logout happens, so that the id, userName + usersRatings get set to nothing